interpreter/interpreter_test.go

```diff
@@ -30,6 +30,51 @@ import (
 	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 )
 
+func TestCompleteInterpreter_ConditionalExpr(t *testing.T) {
+	// a ? b < 1.0 : c == ["hello"]
+	program := NewCompleteProgram(
+		test.Conditional.Expr,
+		test.Conditional.Info(t.Name()))
+
+	interpretable := interpreter.NewInterpretable(program)
+	result, state := interpretable.Eval(
+		NewActivation(map[string]interface{}{
+			"a": true,
+			"b": 0.999,
+			"c": types.NewStringList([]string{"hello"})}))
+	ev, _ := state.Value(6)
```

Comment:
Please comment that the identifier for `c` has the expression id `6`.  Same for the later tests where the magic expression identifier is used. You'll want to mention that the next developer should see the test/expr code to figure out what this value should be (or provide a convenient function that gets the expression id for `c` by name so the next person doesn't have to guess).

Reply:
Comments added~ Please let me know if it is sufficient~